audits/performance/first-meaningful-paint.js

```diff
@@ -28,25 +28,32 @@ class FirstMeaningfulPaint {
     return 'Has a good first meaningful paint';
   }
 
+  /**
+   * Audits the page to give a score for First Meaningful Paint.
+   * @see  https://github.com/GoogleChrome/lighthouse/issues/26
+   * @param  {Object} inputs The inputs from the gather phase.
```

Comment:
sooo if we want to really take advantage of the type annotation here we should define what keys the object has and uses here

same with the `@return`